Test ``TestMenuDocs.testAllMenuItemsAreDocumented`` is falling where it shouldn't.

**Describe the bug**
Test ``TestMenuDocs.testAllMenuItemsAreDocumented`` is falling where it shouldn't. 🫠

**To Reproduce**
Steps to reproduce the behavior:
1. Run ``python test.py tests.mainwindow.TestMenuDocs.testAllMenuItemsAreDocumented`` under Windows 10 with russian locale
2. See error  
    ```bash
    WARNING: foo
    testAllMenuItemsAreDocumented (tests.mainwindow.TestMenuDocs.testAllMenuItemsAreDocumented) ... FAIL
    
    ======================================================================
    FAIL: testAllMenuItemsAreDocumented (tests.mainwindow.TestMenuDocs.testAllMenuItemsAreDocumented)
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "blabla/zim-desktop-wiki/tests/mainwindow.py", line 308, in testAllMenuItemsAreDocumented
        self.assertTrue(label in self.manual, 'Menu item "{}" not documented'.format(label))
    AssertionError: False is not true : Menu item "**Back <Alt><Влево>**" not documented
    
    ----------------------------------------------------------------------
    Ran 1 test in 1.137s
    
    FAILED (failures=1)
    
    Wrote test report to test_report.html
    Wrote test_times.csv
    ```

**Expected behavior**
This test should have passed.

**Screenshots**
N/A

**Environment (please complete the following information):**
 - Version: Any
 - OS: Windows, Linux
 - Language: Any except ``en_US``(?)

**Additional context**
```powershell
PS C:\Users\User> systeminfo /fo csv | ConvertFrom-Csv | select OS*, System* | Format-List


OS Name             : Microsoft Windows 10 Enterprise LTSC
OS Version          : 10.0.19044 N/A Build 19044
OS Manufacturer     : Microsoft Corporation
OS Configuration    : Standalone Workstation
OS Build Type       : Multiprocessor Free
System Manufacturer : Micro-Star International Co., Ltd.
System Model        : MS-7D95
System Type         : x64-based PC
System Directory    : C:\Windows\system32
System Locale       : ru;Russian
```

I looked at the source code for ``Gtk.accelerator_get_label``. 
https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/gtk/gtkaccelgroup.c#L1704-1718
https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/gtk/gtkaccellabel.c#L773-935
https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/po/ru.po#L170-173
Apparently it needs a hack to turn back localized string into non-localized one.
Or look for matches not in ``manual/Help/Menu_Items.txt``.

Аfter some more research, I found that this behavior apparently affects Linux too.
```sh
user@user-pc:~$ locale
LANG=de_DE.UTF-8
LANGUAGE=de_DE:ru:en_US:en
LC_CTYPE="de_DE.UTF-8"
LC_NUMERIC=de_DE.UTF-8
LC_TIME=de_DE.UTF-8
LC_COLLATE="de_DE.UTF-8"
LC_MONETARY=de_DE.UTF-8
LC_MESSAGES="de_DE.UTF-8"
LC_PAPER=de_DE.UTF-8
LC_NAME=de_DE.UTF-8
LC_ADDRESS=de_DE.UTF-8
LC_TELEPHONE=de_DE.UTF-8
LC_MEASUREMENT=de_DE.UTF-8
LC_IDENTIFICATION=de_DE.UTF-8
LC_ALL=
user@user-pc:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.6 LTS
Release:	18.04
Codename:	bionic
```
```sh
user@user-pc:blabla/zim-desktop-wiki$ python3 test.py tests.mainwindow.TestMenuDocs.testAllMenuItemsAreDocumented
WARNING: foo
testAllMenuItemsAreDocumented (tests.mainwindow.TestMenuDocs) ... FAIL

======================================================================
FAIL: testAllMenuItemsAreDocumented (tests.mainwindow.TestMenuDocs)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "blabla/zim-desktop-wiki/tests/mainwindow.py", line 304, in testAllMenuItemsAreDocumented
    self.assertTrue(label in self.manual, 'Menu item "{}" not documented'.format(label))
AssertionError: False is not true : Menu item "===== Datei =====" not documented

----------------------------------------------------------------------
Ran 1 test in 0.135s

FAILED (failures=1)

Wrote test report to test_report.html
Wrote test_times.csv
```

--- a/tests/__init__.py
+++ b/tests/__init__.py
@@ -34,6 +34,7 @@ import unittest
 from unittest import skip, skipIf, skipUnless, expectedFailure
 
 
+os.environ['LANGUAGE'] = 'C.UTF-8'
 gettext.install('zim', names=('_', 'gettext', 'ngettext'))
 
 FAST_TEST = False #: determines whether we skip slow tests or not

